Tensile/Source/lib/include/Tensile/Serialization/PlaceholderLibrary.hpp

```diff
@@ -31,7 +31,7 @@
 
 #include <Tensile/MasterSolutionLibrary.hpp>
 #include <Tensile/PlaceholderLibrary.hpp>
-#include <regex>
+#include <fnmatch.h>
```

Comment:
Might need windows-specific code?

Reply:
good catch, I will add support for windows

Reply:
Yes for windows sake as well did you consider what I suggested in the meeting?   Can the requirement be represented in simpler terms.  If you can put it in words first, it may be just two calls: strstr and maybe a strncmp suffice for what you need.

Reply:
Addressed windows version for pattern matching. @TorreZuk , Sorry I missed this comment. For the current use case, I think it could be done with just strstr. 
- strstr(lib_filename , ArchName) for arch specific 
- strstr(lib_filename, "") for loading all dat files.

My thought process to replace it with an equivalent pattern matching is, if lazy loading is expanded to load datatype specific or something like that, it would be easier to do it with pattern matching. 

I could use strstr if that is preferred.

Reply:
if !strlen(pattern) you load them all so that is simple.  Yes if you expect to need a full glob later then strstr may get too tricky.  The way you link did it bring in a .dll or a static library for sh?   We then static link this into rocblas.dll so confirm rocblas dependencies on windows and if it passes tests that is good enough for me.  Just saying it appears there is variation in consuming this library on windows so something to check out.